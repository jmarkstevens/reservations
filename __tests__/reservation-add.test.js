import 'react-native'
import React from 'react'
// import { expect as expectChai } from 'chai'
import { shallow } from 'enzyme'
import ReservationAddComponent from '../src/components/reservations/reservation-add'

describe('<ReservationAddComponent>', () => {
  const Component = <ReservationAddComponent />

  const wrapper = shallow(Component)

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
