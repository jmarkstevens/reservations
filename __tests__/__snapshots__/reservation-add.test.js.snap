// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReservationAddComponent> matches snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
